src/elements/kw-social-comment/kw-social-comment.html

```diff
@@ -159,18 +169,21 @@ <h3 hidden$="{{comments.length > 0}}">Be the first to comment!</h3>
                 comments: {
                     type: Array,
                     value:[]
-                },
-                share: {
-                    type: Object
                 }
             },
+             _computeHidden(){
```

Comment:
computeHidden is too generic, someone new to the project would not understand what it does